src/api-v1.js

```diff
@@ -669,7 +669,10 @@ api.declare({
       data: secret.secrets,
       scopes: secret.scopes,
```

Comment:
@jhford, nvm, I assume you've planned some cleanup work to remove the workerType.scopes property later...